docs/guides/hooks-list.rst

```diff
@@ -186,6 +182,30 @@ Action hooks
 **forward, <reason>**
 	Filter the URL to forward a user to when ``forward($url, $reason)`` is called.
 
+.. _guides/hooks-list#ajax:
+
+Ajax
+====
+
+**ajax_response, \***
+	When the ``elgg/Ajax`` AMD module is used, this hook gives access to the response object
+	(``\Elgg\Services\AjaxResponse``) so it can be altered/extended. The hook type depends on
+	the method call:
+
+	================  ====================
+	elgg/Ajax method  plugin hook type
+	================  ====================
+	performAction()   action:<action_name>
+	fetchPath()       path:<url_path>
+	fetchView()       view:<view_name>
+	fetchForm()       form:<action_name>
+	================  ====================
```

Comment:
Maybe I should just name the methods like the hook types: action(), path(), view(), form() ...